modin/pandas/dataframe.py

```diff
@@ -2555,7 +2528,7 @@ def __round__(self, decimals=0):
         -------
         DataFrame
         """
-        return self._default_to_pandas(pandas.DataFrame.__round__, decimals=decimals)
+        return self.round(decimals)
```

Comment:
Does this need `self._query_compiler.round`?

Reply:
I think API level round round should work.